go/teleportr/drivers/disburser/driver.go

```diff
@@ -0,0 +1,518 @@
+package disburser
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"math"
+	"math/big"
+	"strings"
+	"time"
+
+	"github.com/ethereum-optimism/optimism/go/bss-core/metrics"
+	"github.com/ethereum-optimism/optimism/go/bss-core/txmgr"
+	"github.com/ethereum-optimism/optimism/go/teleportr/bindings/deposit"
+	"github.com/ethereum-optimism/optimism/go/teleportr/bindings/disburse"
+	"github.com/ethereum-optimism/optimism/go/teleportr/db"
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/ethereum/go-ethereum/log"
+)
+
+// stateRootSize is the size in bytes of a state root.
+const stateRootSize = 32
+
+var bigOne = new(big.Int).SetUint64(1) //nolint:unused
```

Comment:
Did you mean to leave this even though its unused?

Reply:
good catch, was left over from copying in from the BSS driver as a template.